.mandatory-checkstyle.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0"?>
```

Comment:
I'd prefer to move this file to a more visible and standard location. `src/main/config` used to be a [natural location](https://web.archive.org/web/20150108171525/http://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html), even though the [current Maven Standard Directory Layout](https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html) does not mention this directory any more.